schema/base.sql

```diff
@@ -1257,7 +1257,7 @@ CREATE TABLE mail_list (
 COMMENT ON TABLE mail_list IS 'Definitions of mailing lists';
 COMMENT ON COLUMN mail_list.listid IS 'Surrogate Key';
 COMMENT ON COLUMN mail_list.listname IS 'Name of the list';
-COMMENT ON COLUMN mail_list.defn IS 'A SQL string that returns fname, sname and email address.';
+COMMENT ON COLUMN mail_list.defn IS 'A SQL string that returns fname, nname ,sname and email address.';
```

Comment:
if this is correct, won't it break all existing queries in the database?

Reply:
Likely this is true, as I didn't end up using nname names anywhere near mail

Reply:
Will be reverted